.github/workflows/cut-release-branch.yaml

```diff
@@ -0,0 +1,283 @@
+# **what?**
+# Cuts the `*.latest` branch, bumps dependencies on it, cleans up all files in `.changes/unreleased`
+# and `.changes/previous verion on main and bumps main to the input version.
+
+# **why?**
+# Clean up the main branch after a release branch is cut and automate cutting the release branch.
+# Generally reduces the workload of engineers and reducing error.
+
+# **when?**
+# This will run when called manually or when triggered in another workflow.
+
+# Example Usage including required permissions:  TODO: update once finalized
+
+# permissions:
```

Comment:
I'm assuming this needs cleaned up

Reply:
It does.  Trying to get this into main so I can start testing it and then finalize the docs for it.